uint32 min_area_size
-------------------
geometry_msgs/Point[] frontier_points